algorithms/kernel/logistic_regression/logistic_regression_train_dense_default_impl.i

```diff
@@ -59,8 +59,6 @@ services::Status TrainBatchKernel<algorithmFPType, method, cpu>::compute(const H
                                                                          const NumericTablePtr & y, logistic_regression::Model & m, Result & res,
                                                                          const Parameter & par)
 {
-    DAAL_ITTNOTIFY_SCOPED_TASK(compute);
```

Comment:
I see a lot of deletions of tasks inside your changes. Why did you decide to do that? I think this will be a problem when we will collect data for new reports, which need to be aligned with older ones